tasks/10_remove-whitespaces/remove-whitespaces.js

```diff
@@ -0,0 +1,5 @@
+function trimWhiteSpaces(str) {
+  return str.split(' ').join('');
+}
```

Comment:
As Victor said, you can export it immediately, like `export function ...`

Reply:
Yes I can, but I don't have to, right? Or is there a recommended way to export?

Reply:
Yep, you don't have to.

I suggest it because of next points:
1) using this approach you don't understand that function which you looking at is a part of public api or not - you have to scroll down and take a look at exports.
2) using this approach you have to spend time adding each new public function to exports, it would be a bit uncomfortable in a case of huge utilities files, for example.